src/sidebar/components/GroupList/GroupListItem.js

```diff
@@ -48,6 +50,7 @@ function GroupListItem({
     store.clearDirectLinkedGroupFetchFailed();
     store.clearDirectLinkedIds();
     groupsService.focus(group.id);
+    frameSync.notifyNotebook('groupChanged', group.id);
```

Comment:
The focused group can change as a result of other events as well - for example when the logged-in user changes. One way to handle this is to have a service watch `store.focusedGroup()` and make this call in response.